src/briefcase/integrations/android_sdk.py

```diff
@@ -733,34 +735,32 @@ def start_emulator(self, avd):
     https://developer.android.com/studio/run/emulator-acceleration#accel-vm
 """.format(cmdline=' '.join(str(arg) for arg in emulator_popen.args)))
 
-                for device, details in sorted(self.devices().items()):
-                    # Only process authorized devices that we haven't seen.
-                    if details['authorized'] and device not in known_devices:
-                        adb = self.adb(device)
-                        device_avd = adb.avd_name()
-
-                        if device_avd == avd:
-                            # Found an active device that matches
-                            # the AVD we are starting.
-                            full_name = "@{avd} (running emulator)".format(
-                                avd=avd,
-                            )
-                            break
-                        else:
-                            # Not the one. Zathras knows.
-                            adb = None
-                            known_devices.add(device)
-
-                # Try again in 2 seconds...
-                self.sleep(2)
-
-            # Print a marker so we can see the phase change
-            print(' booting...', flush=True, end='')
+                    for device, details in sorted(self.devices().items()):
+                        # Only process authorized devices that we haven't seen.
+                        if details['authorized'] and device not in known_devices:
+                            adb = self.adb(device)
+                            device_avd = adb.avd_name()
+
+                            if device_avd == avd:
+                                # Found an active device that matches
+                                # the AVD we are starting.
+                                full_name = "@{avd} (running emulator)".format(
+                                    avd=avd,
+                                )
+                                break
+                            else:
+                                # Not the one. Zathras knows.
+                                adb = None
+                                known_devices.add(device)
+
+                    # Try again in 2 seconds...
+                    self.sleep(2)
 
             # Phase 2: Wait for the boot process to complete
-            while not adb.has_booted():
-                if emulator_popen.poll() is not None:
-                    raise BriefcaseCommandError("""
+            with self.command.input.wait_bar(" booting...") as boot_wait_bar:
```

Comment:
Really cosmetic; but the space before ` booting` existed previously because it was all on one line; we can drop the space here because it's the start of a new line. It should also be capitalized.

Reply:
Updated.